Slack.Automation/Promact.Erp.Core/Controllers/TaskReportController.cs

```diff
@@ -185,7 +185,7 @@ public TaskReportController(ITaskMailRepository taskMailReport, IStringConstantR
         public async Task<List<TaskMailUserAc>> TaskMailDetailsReportSelectedDate(string UserRole, string CreatedOn, string UserId, string UserName,string SelectedDate)
```

Comment:
This part is belong to @iamronakshah . So he had made changes in his issue #122 .